CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 - Retry `berkshelf` installation up to 3 times.
   
 **BUG FIXES**
+- Fix to not enable slurmd service prior to finalize stage, which might cause user to unintentionally start slurmd and make compute node available in post-install process.
```

Comment:
I see this more as an enhancement than a bug fix

Reply:
Done